### PR TITLE
Dawn - Add B2B only nav

### PR DESCRIPTION
**Pull request by cm-sl**
[Based off this](https://docs.sparklayer.io/shopify-customisations#CH_Mp)

### CODE REVIEW

**Review by timmw — APPROVED** [review]

File: snippets/header-dropdown-menu.liquid [hunk 1]
```diff
@@ -7,7 +7,22 @@
...
+          {% assign menu = linklists[section.settings.menu] %}
+          
+        {%- endif -%}
+
+    {%- else -%}
+
+    {% assign menu = linklists[section.settings.menu] %}
+    {%- endif -%}
+
+    {%- for link in menu.links -%}
```
Comment (rlweb) [inline snippets/header-dropdown-menu.liquid]:
```suggestion
    {%- if customer.metafields.sparklayer.authentication -%}
      {% assign menu = linklists.b2b-menu | default: linklists[section.settings.menu] %}
    {%- else -%}
      {% assign menu = linklists[section.settings.menu] %}
    {%- endif -%}
    {%- for link in menu -%}
```
Simples, if the b2b menu doesn't exist, fall back to default menu, needs testing though, see [here](https://shopify.github.io/liquid/filters/default/) how to use

**Review by rlweb — CHANGES_REQUESTED** [review]

**Review by jam3sn — APPROVED** [review]

File: snippets/header-drawer.liquid [hunk 2]
```diff
@@ -21,7 +21,22 @@
         <div class="menu-drawer__navigation-container">
           <nav class="menu-drawer__navigation">
             <ul class="menu-drawer__menu has-submenu list-menu" role="list">
-              {%- for link in section.settings.menu.links -%}
+                  {% assign showB2BNav = false %}
```
Comment (timmw) [inline snippets/header-drawer.liquid]: Is the expectation that people will change this to true by editing the theme if they want the b2b nav?

Comment (cm-sl) [reply snippets/header-drawer.liquid]: Yes

**Review by LukeSparkLayer — APPROVED** [review]

**Comment by cm-sl** [comment]
Code updated if you can re-review

**Review by LukeSparkLayer — APPROVED** [review]

**Review by jam3sn — APPROVED** [review]

**Pull request merged**